packages/js/client/src/__tests__/e2e/wasm-as.spec.ts

```diff
@@ -137,12 +137,18 @@ describe("wasm-as test cases", () => {
   });
 
   it("implementations - getImplementations", async () => {
+    const interfacePath = `${GetPathToTestWrappers()}/wasm-rs/implementations/test-interface`;
     const interfaceUri = "wrap://ens/interface.eth";
 
-    const implementationPath = `${GetPathToTestWrappers()}/wasm-as/implementations/test-use-getImpl`;
-    const implementationUri = `wrap://fs/${implementationPath}/build`;
+    const implementationPath = `${GetPathToTestWrappers()}/wasm-rs/implementations/test-wrapper`;
```

Comment:
It looks like there is a typo here. The wrapper URIs point to the `wasm-rs` folder, but they should point to `wasm-as`.